packages/collections/map.pony

```diff
@@ -159,6 +169,9 @@ class HashMap[K, V, H: HashFunction[K] val]
     Returns the value, the same as `insert`, allowing 'insert_if_absent'
     to be used as a drop-in replacement for `insert`.
     """
+    let value' = value
+    let value'' = value
```

Comment:
You created two aliases here, but you only used one - did you forget to remove the other when you refactored some code?